src/FakeItEasy.Analyzer/Properties/AssemblyInfo.cs

```diff
@@ -1,11 +1,10 @@
 ﻿using System;
-using System.Reflection;
 using System.Runtime.CompilerServices;
 
-[assembly: AssemblyProduct("FakeItEasy.Analyzer.VisualBasic")]
-[assembly: AssemblyTitle("FakeItEasy.Analyzer.VisualBasic")]
-[assembly: AssemblyDescription("Diagnostic analyzers to warn about incorrect usage of FakeItEasy in VB.NET code.")]

```

Comment:
Nope. The product and title are inferred from the assembly name, and the description is in the csproj.